tests/unittests/test_net.py

```diff
@@ -4576,40 +4576,46 @@ def test_select_none_found_raises(self, m_eni_avail, m_sysc_avail):
                           priority=['sysconfig', 'eni'])
 
     @mock.patch("cloudinit.net.renderers.netplan.available")
+    @mock.patch("cloudinit.net.renderers.sysconfig.available")
     @mock.patch("cloudinit.net.renderers.sysconfig.available_sysconfig")
     @mock.patch("cloudinit.net.renderers.sysconfig.available_nm")
     @mock.patch("cloudinit.net.renderers.eni.available")
     @mock.patch("cloudinit.net.renderers.sysconfig.util.get_linux_distro")
     def test_sysconfig_selected_on_sysconfig_enabled_distros(self, m_distro,
                                                              m_eni, m_sys_nm,
                                                              m_sys_scfg,
+                                                             m_sys_avail,
                                                              m_netplan):
         """sysconfig only selected on specific distros (rhel/sles)."""
 
         # Ubuntu with Network-Manager installed
-        m_eni.return_value = False       # no ifupdown (ifquery)
-        m_sys_scfg.return_value = False  # no sysconfig/ifup/ifdown
-        m_sys_nm.return_value = True     # network-manager is installed
-        m_netplan.return_value = True    # netplan is installed
+        m_eni.return_value = False        # no ifupdown (ifquery)
+        m_sys_scfg.return_value = False   # no sysconfig/ifup/ifdown
+        m_sys_nm.return_value = True      # network-manager is installed
+        m_netplan.return_value = True     # netplan is installed
         m_distro.return_value = ('ubuntu', None, None)
+        m_sys_avail.return_value = False  # no sysconfig on Ubuntu
```

Comment:
A nit, but this line (and its comment) would read more easily if it were a line further up.